src/Phinx/Config/Config.php

```diff
@@ -143,7 +143,7 @@ public function getEnvironments()
     /**
      * {@inheritdoc}
      */
-    public function getEnvironment($name)
+    public function getEnvironment($name, $database = null)
```

Comment:
These changes for multi-database should be a separate PR from adding new column types, as the two are completely unrelated.